features-support/step_definitions/parsing.js

```diff
@@ -29,7 +32,12 @@ module.exports = function() {
 
   function compareFeatureValues(key) {
     return function compare(table) {
-      var featureValues = features[0][key];
+      var featureValues = features[key].map(function(a) {
+        if(key == "comments")
```

Comment:
Rule of thumb in the JS community, either single line ifs with no brackets or use brackets on multilines :smile: